### PR TITLE
Update cloud build for manifestlist + cosign

### PR DESCRIPTION
**Pull request by loosebazooka**
Signed-off-by: Appu Goundan <appu@google.com>

### CODE REVIEW

**Review by dlorenc — APPROVED** [review]

File: cloudbuild_cosign.sh [hunk 1]
```diff
@@ -9,15 +9,19 @@ export KMS_VAL=gcpkms://projects/$PROJECT_ID/locations/global/keyRings/cosign/cr
 cosign version
 
 # Sign images with cosign
-for distro_suffix in "" -debian10; do
+for distro_suffix in "" -debian10 -debian11; do
   cosign sign -key $KMS_VAL gcr.io/$PROJECT_ID/static${distro_suffix}:nonroot
   cosign sign -key $KMS_VAL gcr.io/$PROJECT_ID/static${distro_suffix}:latest
+  cosign sign -key $KMS_VAL gcr.io/$PROJECT_ID/static${distro_suffix}:debug-nonroot
+  cosign sign -key $KMS_VAL gcr.io/$PROJECT_ID/static${distro_suffix}:debug
```
Comment (loosebazooka) [inline cloudbuild_cosign.sh:16]: @dlorenc @dekkagaijin was there a reason these static images weren't previously signed?

Comment (dlorenc) [reply cloudbuild_cosign.sh:16]: @priyawadhwa might know, but I don't think so.

File: cloudbuild_cosign.sh [hunk 2]
```diff
@@ -9,15 +9,19 @@ export KMS_VAL=gcpkms://projects/$PROJECT_ID/locations/global/keyRings/cosign/cr
...
 
   cosign sign -key $KMS_VAL gcr.io/$PROJECT_ID/base${distro_suffix}:nonroot
   cosign sign -key $KMS_VAL gcr.io/$PROJECT_ID/base${distro_suffix}:latest
   cosign sign -key $KMS_VAL gcr.io/$PROJECT_ID/base${distro_suffix}:debug-nonroot
   cosign sign -key $KMS_VAL gcr.io/$PROJECT_ID/base${distro_suffix}:debug
+done
 
+for distro_suffix in "" -debian10; do
```
Comment (evanj) [inline cloudbuild_cosign.sh:24]: Does this mean the `cc` and `python` images are not being supported for `-debian11`, or is this "temporary"?

Comment (loosebazooka) [reply cloudbuild_cosign.sh:24]: This is temporary, we're upgrading in phases.

File: cloudbuild_cosign.sh (hunk 1 above)
Comment (priyawadhwa) [reply cloudbuild_cosign.sh:16]: no reason! i probably forgot to include them 😓

**Pull request merged**